src/modules/host/host-list/host-list.html

```diff
@@ -76,7 +76,7 @@ <h5 ng-if="hostCntrl.hostList.length !== 0">{{hostCntrl.filteredHostList.length}
         </center><div ng-if="hostCntrl.isDataLoading" class="dataLoading"></div>
         <div ng-if="!hostCntrl.isDataLoading" class="list-group list-view-pf list-view-pf-equalized-column" ng-repeat="host in hostCntrl.filteredHostList=(hostCntrl.hostList | filter: hostCntrl.searchBy | orderBy : hostCntrl.ascOrder + hostCntrl.orderBy)">
             <div class="list-group-item">
-                <div class="list-view-pf-left col-md-2">
+                <div class="col-md-2" ng-class="{ 'list-view-pf-left': clusterId === undefined }">
```

Comment:
@kamleshverma1 Please use typeof operator to compare with undefined. I have told you the reasons to use typeof in previous PR